Automattic/wp-calypso
# Experiment: Restructure /me/security

#### Changes proposed in this Pull Request

* Push the new checklist format from #43101 to be the main nav for `/me/security`, which includes a split for actionable items versus informational ones. (Th
* To handle the above, push the change password page to `/me/security/password`.
* Restructure the individual pages under `/me/security` to use the `HeaderCake` component to provide a heading with a "Back" button to get back to the checklist at `/me/security`. (This isn't implemented for the account settings, where we change email.)

##### Open Questions
* How should we handle the password link/text until we can actually implement the infrastructure to work out when the password was last changed.

#### Testing instructions

* Navigate to the live build at https://calypso.live/?branch=try/me-security-restructure
* After you get redirected to the current UI, check that `/me/security` works as before
* Manually add `?flags=security/security-checkup` to the URL, and navigate to `/me/security/`.
* For each of the links, verify the following:
    - The link works
    - The page you navigate to includes a header with a "Back" button
    - The "Security" nav item on the left nav bar is selected for the page
    - Clicking on the "Back" button brings you back to the checklist at `/me/security` (except for the email row, as that navigates to `/me/account`)

Fixes #43144
